iroha-cli/interactive/impl/interactive_transaction_cli.cpp

```diff
@@ -395,6 +392,15 @@ namespace iroha_cli {
       return generator_.generateSetQuorum(account_id, quorum.value());
     }
 
+    std::shared_ptr<Command>
+    InteractiveTransactionCli::parseSetAccountDetail(
+        std::vector<std::string> params) {
+      auto account_id = params[0];
+      auto key = params[1];
+      auto value = params[2];
+      return generator_.generateSetAccountDetail(account_id, key, value);
```

Comment:
I think there is no need to make new generate function, might be better to use just constructor of the function